crates/biome_service/src/settings.rs

```diff
@@ -953,7 +953,6 @@ pub struct OverrideSettingPattern {
     pub(crate) cached_json_format_options: RwLock<Option<JsonFormatOptions>>,
     pub(crate) cached_css_format_options: RwLock<Option<CssFormatOptions>>,
     pub(crate) cached_js_parser_options: RwLock<Option<JsParserOptions>>,
-    pub(crate) cached_json_parser_options: RwLock<Option<JsonParserOptions>>,
```

Comment:
I wonder if we could keep it, add a `_` at the beginning of its name, and add `#[allow(dead_code)]`. I am sure we will need it again once we fix the caching